pkg/transforms/encryption.go

```diff
@@ -0,0 +1,89 @@
+//
+// Copyright (c) 2017 Cavium
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package transforms
+
+import (
+	"bytes"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/sha1"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/appcontext"
+)
+
+type Encryption struct {
+	Key string
+	Iv  string
```

Comment:
Can we use InitVector for the name? ;-)

Reply:
we can though, IV is pretty standard notation, and I know when I see IV what the means, where as InitVector i wouldnt relate it as much since I never see it that way. It is InitVector in the models.EncryptionDetails which is ultimately what is used by the end-user.

Reply:
Don't forget maintainers after we have long moved on... I took me a while to figure out what Iv meant and I have done encryption stuff in the past.